samples/index.html

```diff
@@ -11,9 +11,9 @@
   <div id="header"><h1><a href="/">WebCodecs API</a></h1></div>
   <div id="wrapper">
     <div id="content">
-    <article onclick="window.location.href='/';">
-      <h1><a href="/">Sample Demo 1</a></h1>
-      <p>A sample that shows cool WebCodecs stuff.</p>
+    <article onclick="window.location.href='/samples/mp4-decode/';">
+      <h1><a href="/samples/mp4-decode/">MP4 Decoding</a></h1>
+      <p>Demuxes (using mp4box.js) and decodes an mp4 file.</p>
     </article>
     <article onclick="window.location.href='/';">
```

Comment:
I'd just delete the rest of these place holder ones now that there's a real entry :)

Reply:
Actually since we're all working on this never mind, easier to merge multiple commits :)

Reply:
Seems good.

Reply:
Hrm. Deleting a commit with Windows Github Desktop is not as easy as I hoped...

Reply:
I created a new branch. I'm still haven't fixed my main though.